api/README.md

```diff
@@ -0,0 +1,74 @@
+# MongoDB Atlas Versioned SDK
+
+A Go HTTP client for the [MongoDB Atlas API](https://docs.atlas.mongodb.com/api/)
+
+## Usage
+
+### Adding Dependency 
+
+```
+go install go.mongodb.org/atlas
+```
+
+### Using in the code
+
+```go
+mongodbatlas import "go.mongodb.org/atlas/mongodbatlas/api/v1"
+```
+
+Construct a new Atlas client, then use the various services on the client to
+access different parts of the Atlas API. For example:
+
+```go
+	mongodbatlas import "go.mongodb.org/atlas/mongodbatlas/api/v1"
+
+   	apiKey := os.Getenv("MDB_API_KEY")
+	apiSecret := os.Getenv("MDB_API_SECRET")
+
+	// SDK client with authentication
+	sdk := mongodbatlas.NewSDKClientWithCredentials(apiKey, apiSecret)
+    projects, response, err := sdk.ProjectsApi.ListProjects(ctx).Execute()
```

Comment:
seems like a small indentation issue here, ie spaces vs tabs

Reply:
Good catch. 
I will consider setting up prettier in the repository to catch up issues like this.